src/components/CollectionsAndPalettes/common/CollectionsHeader/CollectionsHeader.module.scss

```diff
@@ -0,0 +1,76 @@
+@import "styles/global";
+@import "styles/variables.scss";
+
+.Header {
+  background-color: black;
+  display: flex;
+  flex-direction: row;
+  color: white;
+  padding: $margin-small-4 $margin-huge-5;
+  column-gap: $margin-medium-3;
+}
+.headerLogo {
+  h1 {
+    font-size: 26px;
+    color: white;
+  }
+  div {
+    display: flex;
+    align-items: center;
+    margin-right: 0 !important;
+    svg {
+      height: 40px;
+
+      path {
+        fill: white;
+      }
+    }
+  }
+}
+.navLinks {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  column-gap: $margin-medium;
+
+  .navLink {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+
+    a {
+      color: rgba(#fff, 0.8);
+      font-size: var(--font-size-lg);
+      padding-right: 10px;
+      cursor: pointer;
+      transition: color 0.5s;
```

Comment:
I think those transitions should be a bit faster, maybe try 0.2s. It feels a bit unresponsive right now.